File: ambeth/Ambeth.IoC/ambeth/ioc/threadlocal/IThreadLocalCleanupBeanExtendable.cs
namespace De.Osthus.Ambeth.Ioc.Threadlocal
{
    public interface IThreadLocalCleanupBeanExtendable
    {
        void RegisterThreadLocalCleanupBean(IThreadLocalCleanupBean threadLocalCleanupBean);

        void UnregisterThreadLocalCleanupBean(IThreadLocalCleanupBean threadLocalCleanupBean);
    }
}

File: jambeth/jAmbeth.IoC/src/de/osthus/ambeth/ioc/threadlocal/IThreadLocalCleanupBeanExtendable.java
package de.osthus.ambeth.ioc.threadlocal;

public interface IThreadLocalCleanupBeanExtendable
{
	void registerThreadLocalCleanupBean(IThreadLocalCleanupBean threadLocalCleanupBean);

	void unregisterThreadLocalCleanupBean(IThreadLocalCleanupBean threadLocalCleanupBean);
}